src/collections/componentSetBuilder.ts

```diff
@@ -123,7 +123,7 @@ export class ComponentSetBuilder {
         logger.debug(`Building ComponentSet from targetUsername: ${org.username}`);
         const fromConnection = await ComponentSet.fromConnection({
           // eslint-disable-next-line @typescript-eslint/no-unsafe-call,@typescript-eslint/no-unsafe-member-access,@typescript-eslint/no-unsafe-assignment
-          usernameOrConnection: (await Aliases.fetch(org.username)) || org.username,
+          usernameOrConnection: (await GlobalInfo.create()).aliases.get(org.username) || org.username,
```

Comment:
confirm that the new `aliases.get` returns something falsy
prefer ?? over || if it's returning undefined/null

Reply:
@mshanemc @WillieRuemmele as in this case `org.username` is either an username or an alias we can use here ```usernameOrConnection: (await GlobalInfo.create()).aliases.resolveUsername(org.username),```